test/js/node/http/node-http.test.ts

```diff
@@ -1804,4 +1806,33 @@ it("#10177 response.write with non-ascii latin1 should not cause duplicated char
       }
       finish();
     });
+}, 20_000);
+
+it("should emit events in the right order", async () => {
+  const { stdout, stderr, exited } = Bun.spawn({
+    cmd: [bunExe(), "run", path.join(import.meta.dir, "fixtures/log-events.mjs")],
+    stdout: "pipe",
+    stdin: "pipe",
+    stderr: "pipe",
+    env: bunEnv,
+  });
+  expect(stderr).toBeDefined();
```

Comment:
we shouldn't need to check that stderr/stdout is defined, that would be a bug in Bun.spawn if it weren't

```suggestion
  expect(await exited).toBe(0);
```